### PR TITLE
[4.0] Check if rules are available in rules field rule

### PR DESCRIPTION
**Pull request by laoneo**
Pull Request for Issue #23115.

### Summary of Changes
Check if the rules from the access.xml are available.

### Testing Instructions
- Go to Global Configuration
- Click Save button
- View PHP error log file

### Expected result
All is working as expected.

### Actual result
The following error is written to the server log:
_Invalid argument supplied for foreach()_

### CODE REVIEW

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on d21144fc0057b51fbcac88823ad8953239ddbed9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23125">issues.joomla.org/tracker/joomla-cms/23125</a>.</sub>

File: libraries/src/Form/Rule/RulesRule.php [hunk 1]
```diff
@@ -100,10 +100,12 @@ protected function getFieldActions(\SimpleXMLElement $element)
 			"/access/section[@name='" . $section . "']/"
 		);
 
-		// Iterate over the asset actions and add to the actions.
-		foreach ($elActions as $item)
+		if ($elActions)
 		{
-			$actions[] = $item->name;
+			// Iterate over the asset actions and add to the actions.
+			foreach ($elActions as $item) {
```
Comment (SharkyKZ) [inline libraries/src/Form/Rule/RulesRule.php]:
```suggestion
			foreach ($elActions as $item)
			{
```

Comment (infograf768) [reply libraries/src/Form/Rule/RulesRule.php]: agree

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on a2123fa8ff4fbe803853e5bfc39b9c80254ea59a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23125">issues.joomla.org/tracker/joomla-cms/23125</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on a2123fa8ff4fbe803853e5bfc39b9c80254ea59a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23125">issues.joomla.org/tracker/joomla-cms/23125</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23125">issues.joomla.org/tracker/joomla-cms/23125</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**